docs/tutorials/alibabacloud.md

```diff
@@ -144,7 +144,7 @@ metadata:
   name: external-dns
 rules:
 - apiGroups: [""]
-  resources: ["services"]
+  resources: ["services","endpoints"]
```

Comment:
If you adding those in the same line as services I would also add pods into it.

Reply:
Done for all tutorials.